src/actions/actions.js

```diff
@@ -10,26 +10,22 @@ export const getTeams = () => async dispatch => {
     dispatch({ type: 'GET_TEAM_COLOR', payload: raptorsTeam.primaryColor });
 
     const raptorsRosterResponse = await dataNbaNet.get('/json/cms/noseason/team/raptors/roster.json');
-    dispatch({ type: 'GET_PLAYERS', payload: raptorsRosterResponse.data.sports_content.roster.players.player });
+    const raptorsRoster = raptorsRosterResponse.data.sports_content.roster.players.player.map((player) => {
```

Comment:
There was a bug, the team color would linger for a second even after the user switches teams.  So I added a property to the player object, "teamColor," which made it easier to access the team color and fixed the bug.